### DISCUSSION TITLE
QOL: Show empty quotas to monitor unlimited mbox sizes

### DISCUSSION THREAD

**Discussion by phpony** [modoboa/modoboa#2955] [Ideas]
There's no way to see how much of space is used by mbox if there's no quota on it. But sometimes it's just useful to know the size of certain box.

On my local instance I've commented out this line:
https://github.com/modoboa/modoboa/blob/f39052b8d0f4f311f31b5e132bb9fd1f9bdf62c7/modoboa/admin/views/identity.py#L74

And changed this line:
https://github.com/modoboa/modoboa/blob/0589c665f2aae86658f977bee939390fc16a2746/modoboa/admin/templates/admin/identities_quotas.html#L7

To this code:
```
    {% if mbox.quota == 0 %}
	unlim
    {% else %}
	{{ mbox.quota }}M
    {% endif %}
```

This small change is a big QOL improvement because now the "quotas" tab is also working as a "disk usage" tab, including all boxes on domain and not only limited ones.

Maybe this can be added to next version?

**Comment by Spitfireap** [comment]
Hi, thanks for the input! We are migrating to new interface, but would you be able to create a PR with these changes?
